README.md

```diff
@@ -1,39 +1,82 @@
-# YOURPROJECTNAME
+# Blackbox HelloWorld Responder
 
-Describe what this project is all about.
+The blackbox-helloworld-responder's sole purpose is to respond with "Hello World!" on either a HTTP or TCP connection. It is intended to work as a simple responder for blackbox or synthetic transaction monitoring, using tools like the [Prometheus Blackbox Exporter](https://github.com/prometheus/blackbox_exporter). It purposely is written very simple, so that it can run securely and with minimal resources.
 
-## Installation
+## License
 
-Add steps for installing here.
+See [LICENSE](LICENSE) for details.
 
-## Documentation
+## Code of Conduct
 
-Add documentaiton here, or add a link to
-the full documentation.
+See [CODE_OF_CONDUCT.md](.github/CODE_OF_CONDUCT.md)
+for details.
 
-## Developing
+# Execution
 
-See [CONTRIBUTING.md](.github/CONTRIBUTING.md)
-for details.
+The executable supports overwriting of the ports via environment variables and will log all connections to stdout.
 
-## License
+## Settings
 
-See [LICENSE](LICENSE) for details.
+| Protocol | Default Port | Environment Variable |
+| --- | --- | --- |
+| HTTP | 8080 | HELLO_WORLD_HTTP_PORT |
+| TCP | 8081 | HELLO_WORLD_TCP_PORT |
 
-**START REMOVE THIS BEFORE RELEASE OF PROJECT**
+## Sample process output
+```
+./blackbox-helloworld-responder
 
-Each source file must include a header comment
-with the Apache 2.0 license details. The copyright
-section in the header must be filled in with the below
-details.
+2020/07/28 13:57:39 Starting HTTP Server on 8080
+2020/07/28 13:57:39 Starting TCP Server on 8081
+2020/07/28 13:57:46 HTTP 127.0.0.1:60513
+2020/07/28 13:57:50 TCP 127.0.0.1:60514
+```
 
+## Testing
 ```
-Copyright 2019 Kohl's Department Stores, Inc. 
+# HTTP
+curl -v 127.0.0.1:8080
+
+# TCP
+nc 127.0.0.1 8081
 ```
-**END REMOVE THIS BEFORE RELEASE OF PROJECT**
 
-## Code of Conduct
+# Development
 
-See [CODE_OF_CONDUCT.md](.github/CODE_OF_CONDUCT.md)
-for details.
+## Binary
+
+If you just need a local version to test, then the simplest way is to execute:
+
+```
+make build
+```
+
+## Image
+
+In order to build the docker image, simply execute
+
+```
+make image
+```
+
+## Testing
+
+### Unit Test
+`make test`
+
+## Releasing
+
+This project is using [goreleaser](https://goreleaser.com). GitHub release creation is automated using Travis
+CI. New releases are automatically created when new tags are pushed to the repo.
+```
+$ TAG=v0.0.2 make tag
```

Comment:
Assuming this is locked down permissions wise?

Reply:
Yes you need push access to the repo